Makefile.am

```diff
@@ -38,7 +38,7 @@ dist_doc_DATA = \
 	COPYRIGHT.GPL \
 	COPYING
 
-AM_CPPFLAGS = $(OPENSSL_CRYPTO_CFLAGS) -D_UNICODE
+AM_CPPFLAGS = $(CRYPTO_CFLAGS) -D_UNICODE
```

Comment:
Why this change? How would this work without also changing the prefix OPENSSL_CRYPTO  to CRYPTO  in PKG_CHECK_MODULES test? Also changing the name will break some existing build scripts, especially when cross-compiling.